### ISSUE TITLE
Slow suite count report is invalid

### ISSUE THREAD

**Issue by renchap** [kern/minitest-reporters#270]
After upgrading from 1.3.2 to 1.3.3, the `slow_suite_count` option for the default reporter no longer works:
```
Slowest test classes:

0.000000s String
0.000000s String
0.000000s String
```

**Comment by os97673** [comment]
I think I've fixed it and released as 1.3.4.  Please test and either confirm (preferably ;) or reopen.

**Comment by renchap** [comment]
Thanks! I just tested the new release and the class names are correct, but it looks like the durations are still 0.000000s for all classes:

```
Slowest tests:

0.763171s test_xxx
0.260319s test_xxx
0.207378s test_xxx
0.187944s test_xxx
0.185472s test_xxx

Slowest test classes:

0.000000s xxMailerTest
0.000000s xxControllerTest
0.000000s xxControllerTest
```

It might also be nice to reduce the number of decimals, 3 decimals (milliseconds) should be precise enough.

Unfortunately I cant reopen the issue myself.

**Comment by os97673** [comment]
reopened.  It would be great if you'd wrote a (unit) test for this problem (as PR)  This will simplify my work a lot.

**Comment by os97673** [comment]
added one more part of the fix and release 1.3.5.beta1, please test and let me know if if works for you and your do not see any additional problems.

**Comment by renchap** [comment]
I just tested it and it fixed the problem, thanks 👍

I tried to look at how to write a test for this but havent found the time to dig into this gem's code and understand how to test the report outputs.

Here is the (working) result with `1.3.5.beta1`:
```
Slowest tests:

0.966806s test_xx
0.208932s test_xx
0.193775s test_xx
0.186973s test_xx

Slowest test classes:

5.244817s xxControllerTest
4.290117s xxControllerTest
1.196527s xxControllerTest
```
